src/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProvider.php

```diff
@@ -26,15 +26,21 @@
 
 namespace PrestaShopBundle\Service\DataProvider\Admin;
 
+use PrestaShop\PrestaShop\Adapter\Module\Module as ApiModule;
 use GuzzleHttp\Exception\RequestException;
 use PrestaShopBundle\Service\DataProvider\Marketplace\ApiClient;
 use Psr\Log\LoggerInterface;
+use stdClass;
```

Comment:
I might be wrong but I do not see it being used in this page 😮 ?

Reply:
it is used on line 109.

`     * @param array|stdClass $categoriesListing Category listing`